### PR TITLE
travis: speed up (maint-2.0)

### PR DESCRIPTION
**Pull request by crepererum**
This is the `maint-2.0`-version of #3371

### CODE REVIEW

**Comment by jirikuncar** [comment]
@crepererum please rebase. Thx

File: .travis.yml [hunk 1]
```diff
@@ -41,9 +43,16 @@ python:
 #  - "2.6"
   - "2.7"
 
+addons:
+  apt:
+    packages:
+      - apache2
+      - git
```
Comment (jirikuncar) [inline .travis.yml]: do we need also `subversion`?

Comment (crepererum) [inline .travis.yml]: It was mentioned in .travis.yml before, but honestly I don't think so.

Comment (jirikuncar) [inline .travis.yml]: :ok:

Comment (tiborsimko) [inline .travis.yml]: @crepererum @jirikuncar subversion was (is) needed for the software archiver, see `invenio/utils/vcs/svn.py` and `invenio/testsuite/test_utils_vcs_svn.py` and friends.

**Pull request merged**